apps/dot-cdn/src/app/app.component.ts

```diff
@@ -1,59 +1,152 @@
 import { Component, OnInit, ViewChild } from '@angular/core';
-import { Observable } from 'rxjs';
-import { ChartData, ChartOptions, SelectValues } from './app.interface';
+import { ChartData, ChartOptions, DotCDNStats, SelectValue, DotChartStats } from './app.interface';
+import { DotCDNService } from './dotcdn.service';
+import { FormGroup, FormBuilder } from '@angular/forms';
 
+enum ChartPeriod {
+    Last30Days = '30',
+    Last60Days = '60'
+}
 @Component({
     selector: 'dotcms-root',
     templateUrl: './app.component.html',
     styleUrls: ['./app.component.scss']
 })
 export class AppComponent implements OnInit {
     @ViewChild('chart', { static: true }) chart: any;
-    content$: Observable<{ label: string; value: string }[]>;
-
-    values: SelectValues[] = [
-        { name: 'Last 30 days', value: 'last_30' },
-        { name: 'Last 60 days', value: 'last_60' }
+    purgeZoneForm: FormGroup;
+    periodValues: SelectValue[] = [
+        { name: 'Last 30 days', value: ChartPeriod.Last30Days },
+        { name: 'Last 60 days', value: ChartPeriod.Last60Days }
     ];
-
-    data: ChartData | Record<string, unknown> = {};
+    selectedPeriod: Pick<SelectValue, 'value'> = { value: ChartPeriod.Last30Days };
```

Comment:
Why this is not just ?

```suggestion
    selectedPeriod: ChartPeriod = ChartPeriod.Last30Days;
```

Reply:
Because I only need the `value` member.